src/components/bookItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useDispatch } from "react-redux";
+import { addToCart } from "../../routes/cart/cart.reducer";
+import type { AppDispatch } from "../../store/store";
+import { Book } from "../../types";
+import "./bookItem.styles.css";
```

Comment:
Never write the actual code inside an index file. It is used for exporting elements.
Also, we already had a BookCard component. Why would you create this one. Furthermore, the naming of the component is poor.

Reply:
Have kept the index minimalistic